module/support/configuration/SoccerConfig/data/config/FieldDescription.yaml

```diff
@@ -22,11 +23,11 @@ GoalCrossbarWidth: 0.10
 GoalCrossbarDepth: 0.10
 GoalNetHeight: 1.0
 # Measured from the goal line.
-PenaltyMarkDistance: 1.5 # to the middle of the penalty mark
-CenterCircleDiameter: 1.5
-BorderStripMinWidth: 1.0
+PenaltyMarkDistance: 0 # to the middle of the penalty mark
+CenterCircleDiameter: 1.2
+BorderStripMinWidth: 0.5
 PenaltyRobotStart: 1.0
 
 # New field markings
-PenaltyAreaLength: 2
-PenaltyAreaWidth: 5
+PenaltyAreaLength: 0
+PenaltyAreaWidth: 0
```

Comment:
does this work directly? I added some if statements in the module to make sure it didn't try drawing anything for the penalty area when I tested it the other week.

Reply:
It's fine to leave as 0 without checks.